api/scpca_portal/templates/readme/citation/index.md

```diff
@@ -0,0 +1,24 @@
+## Citation
+
+If you use these data in your research, you must cite:
+
+- The data submitter using language provided as part of the project abstract (as applicable), the publication listed for the project (as applicable), or both.
+- The ScPCA Portal using the language below.
+
+For more information, please see [the How to Cite section of our documentation](https://scpca.readthedocs.io/en/stable/citation.html).
+
+### Citing this project
+
+{% if projects|length > 1 %}
+{% for project in projects %}
+To cite data from {{ project.scpca_id }}, please see the project abstract and publication information at [{{ project.scpca_id }} page.]({{ project.url }})
+{% endfor %}
+{% else %}
+To cite data from {{ projects.0.scpca_id }}, please see the project abstract and publication information at [{{ projects.0.scpca_id }} page.]({{ projects.0.url }})
+{% endif %}
```

Comment:
```suggestion
{% for project in projects %}
To cite data from {{ project.scpca_id }}, please see the project abstract and publication information at [{{ project.scpca_id }} page.]({{ project.url }})
{% endfor %}
```

Reply:
See the commit at 2e7d7b204961c1b889edd80058ae0b71223bb04f.

I've also reduced the additional spaces in the markdown templates to prevent tags (e.g. `for`) from automatically inserting line breaks, thanks for this tip David 👍